### PR TITLE
support bigtable

### PR DESCRIPTION
**Pull request by ericyz**
If this PR is for Terraform, I acknowledge that I have:

- [x] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [x] [Generated Terraform](https://github.com/GoogleCloudPlatform/magic-modules#generating-downstream-tools), and ran `make test` and `make lint` to ensure it passes unit and linter tests.
- [x] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/third_party/terraform/tests) (for handwritten resources or update tests).
- [x] [Ran](https://github.com/hashicorp/terraform-provider-google/blob/master/.github/CONTRIBUTING.md#tests) relevant acceptance tests (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [x] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/master/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:none

```

### CODE REVIEW

File: mmv1/third_party/validator/bigtable_cluster.go [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+	}
+	return assets, nil
+}
+
+func GetBigtableClusterApiObjects(d TerraformResourceData, config *Config) ([]map[string]interface{}, error) {
+	return expandBigtablClusters(d.Get("cluster"), d, config)
+	
+}
+
+func expandBigtablClusters(v interface{}, d TerraformResourceData, config *Config) ([]map[string]interface{}, error) {
```
Comment (melinath) [inline mmv1/third_party/validator/bigtable_cluster.go:42]:
```suggestion
func expandBigtableClusters(v interface{}, d TerraformResourceData, config *Config) ([]map[string]interface{}, error) {
```

**Review by melinath — CHANGES_REQUESTED** [review]
It looks like this needs to have the files added to [mmv1/provider/terraform_object_library.rb](https://github.com/GoogleCloudPlatform/magic-modules/blob/master/mmv1/provider/terraform_object_library.rb) - sorry if I'm jumping the gun on the review due to auto-assignment :-) Otherwise seems reasonable.

File: mmv1/third_party/validator/bigtable_cluster.go (hunk 1 above)
Comment (ericyz) [reply mmv1/third_party/validator/bigtable_cluster.go]: All good mate. Thanks for picking up the typo ;)

**Review by melinath — APPROVED** [review]
LGTM

**Pull request merged**